File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/12/12.go
// Q12: Print repeated numbers per row (1, 22, 333, 4444, 55555).
// Input: An integer n
// Output: Repeated number pattern

package main

import (
    "fmt"
    "strings"
)

func main() {
    var n int
    fmt.Scan(&n)
    for i := 1; i <= n; i++ {
        fmt.Println(strings.Repeat(fmt.Sprintf("%d", i), i))
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/12/12.js
// Q12: Print repeated numbers per row (1, 22, 333, 4444, 55555).
// Input: An integer n
// Output: Repeated number pattern

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 1; i <= n; i++) {
        console.log(String(i).repeat(i));
    }
    rl.close();
});
